app/src/docs/v1.api-spec.yaml

```diff
@@ -169,32 +169,19 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-  "/keyCloak/clients/{keycloakEnv}/{acronym}":
```

Comment:
I know this spec will need tidying up once we are sure about the name for this endpoint